libraries/AP_Notify/Display.cpp

```diff
@@ -452,7 +452,7 @@ void Display::update_gps(uint8_t r)
             fixname = gpsfixname[0];
             break;
     }
-    snprintf(msg, DISPLAY_MESSAGE_SIZE, "GPS:%s Sats:%u", fixname, AP_Notify::flags.gps_num_sats) ;
+    snprintf(msg, DISPLAY_MESSAGE_SIZE, "GPS:%-5s Sats:%2u", fixname, AP_Notify::flags.gps_num_sats) ;
```

Comment:
This doesn't limit the string size, it puts a minimum. Also, what is the point of restricting the string maximum length? If it won't never be shown then short the string. And in the end, Peter added this so that "Sats" always stays in the same place, but if you increase the string size you have to change this anyway.
This can stay, but then the strings should be changed to have only the needed characters instead of useless spaces.